ClassBuilder/serialization/Composite.java

```diff
@@ -0,0 +1,38 @@
+package serialization;
+
+import java.awt.*;

```

Comment:
Wildcard imports, static or otherwise, are not used.
https://mate-academy.github.io/style-guides/java/java.html#s3.3.1-wildcard-imports